src/views/base.html

```diff
@@ -1,53 +1,46 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-  <head>
-    <title>{% block pageTitle %}{{ title }}{% endblock %}</title>
-    <meta name="title" content="{{ title }}" />
-	  <meta name="description" content="Join us for 24 hours of building, breaking and creating in the heart of Cambridge from January 28 – 29." />
-    <meta property="og:title" content="{{ title }}" />
-    <meta property="og:url" content="{{ requestedUrl.href }}" />
-    <meta property="og:description" content="Join us for 24 hours of building, breaking and creating in the heart of Cambridge from January 28 – 29." />
-    <meta property="og:image" content="{{ requestedUrl.protocol }}//{{ requestedUrl.host }}{{ asset('images/fb-share.jpg') }}" />
-
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="language" content="en">
-    <meta name="theme-color" content="{{ colors.PRIMARY_COLOR }}">
-    <script src="https://use.typekit.net/jig8uxt.js"></script>
-    <script>try{Typekit.load({ async: true });}catch(e){}</script>
-    {% if browserSync %}{{ browserSync | safe }}{% endif %}
```

Comment:
Removing this might break the index, terms, privacy etc. pages of our current site — for now I would suggest creating a new base template for HC 2018.  We can then transition over all the old pages to this new template and remove the old template in a new pull request.